packages/bitcore-wallet-service/test/integration/helpers.js

```diff
@@ -553,12 +554,26 @@ helpers.clientSign = function(txp, derivedXPrivKey) {
       // For eth => account, 0, change = 0
       const priv =  xpriv.derive('m/0/0').privateKey;
       const privKey = priv.toString('hex');
-      const rawTx = txp.getBitcoreTx().uncheckedSerialize();
-      signatures = [CWC.Transactions.getSignature({
-        chain: 'ETH',
-        tx: rawTx,
-        key: {privKey},
-      })];
+      const tx = txp.getBitcoreTx().uncheckedSerialize();
+      const isERC20 = txp.tokenAddress && !txp.payProUrl;
+      const chain = isERC20 ? 'ERC20' : ChainService.getChain(txp.coin);
+      if (tx.length === 0) {
```

Comment:
again on the 0 length, why is tx zero length here?

Reply:
changed to `typeof tx === 'string'`